--- a/include/configs/rpi.h	2018-02-08 22:05:32.099689068 +0100
+++ b/include/configs/rpi.h	2018-02-15 22:26:44.577436227 +0100
@@ -149,9 +149,13 @@
 
 #define CONFIG_EXTRA_ENV_SETTINGS \
 	"dhcpuboot=usb start; dhcp u-boot.uimg; bootm\0" \
+	"silent=1\0" \
+	"bootdelay=0\0" \
 	ENV_DEVICE_SETTINGS \
 	ENV_MEM_LAYOUT_SETTINGS \
 	BOOTENV
 
+#define CONFIG_SILENT_CONSOLE
+#define CONFIG_SYS_DEVICE_NULLDEV
 
 #endif
